Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationObserver.cs

```diff
@@ -11,37 +11,35 @@ namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
     /// <summary>
     /// This class observes changes and updates content on a spectator device.
     /// </summary>
-    public class StateSynchronizationObserver : Singleton<StateSynchronizationObserver>
+    public class StateSynchronizationObserver : LocatableDeviceNetworkManager<StateSynchronizationObserver>
```

Comment:
>LocatableDeviceNetworkManager [](start = 48, length = 29)

Is there any instance where a StateSynchronizationBroadcast also needs to report its location?

Reply:
I've refactored the "LocatableDevice" logic into a separate class for the next review, so the StateSynchronizationBroadcaster could in theory also have a LocatableDevice associated with it.